src/testUtil/cleanUpTestCaseCommand.ts

```diff
@@ -1,16 +1,12 @@
 import { TestCaseCommand } from "./TestCase";
+import { merge } from "lodash";
 
 export function cleanUpTestCaseCommand(
   command: TestCaseCommand
 ): TestCaseCommand {
-  const { action, ...rest } = command;
-  const { args } = action;
-
-  return {
-    ...rest,
-    action: {
-      ...action,
-      args: args == null ? undefined : args.length === 0 ? undefined : args,
-    },
-  };
+  const { args } = command.action;
+  const result = merge({}, command);
+  result.action.args =
+    args == null ? undefined : args.length === 0 ? undefined : args;
```

Comment:
This might be clearer?
```suggestion
  if (args == null || args.length === 0) {
    result.action.args = undefined;
  }
```
I'm assuming lodash merge means that `result.action !== command.action` so we're not accidentally mutating it

Reply:
Yea, lodash merge does a deep merge of the objects.

Thanks for pointing out the nested ternary. I'd prefer `result.action.args = !args?.length ? undefined : args;` just because it's a bit shorter, but is that clear enough?

Reply:
On second thought, yours is clearer, let's go with that.